doc/admin-guide/puppet.md

```diff
@@ -0,0 +1,17 @@
+## Automatic setup with Puppet
+
+The Puppet module can be  found [here](https://github.com/indigo-iam/puppet-indigo-iam).
+
+The module provided configures only the IAM Login Service, 
```

Comment:
This module configures the IAM login service packages installation and configuration, and the automatic generation of JSON keystore. 
The setup of the MySQL database used by the service as well as the setup of the reverse proxy are *not covered* by this module.